### PR TITLE
[10.0][MIG] hr_timesheet_activity_begin_end

### PR DESCRIPTION
**Pull request by luismontalba**
Timesheet Activities - Begin/End Hours
-------------------------------------------------
Adds starting and ending hours fields on the timesheet activities.
The new fields are displayed on the timesheet activities.

cc @Tecnativa

### CODE REVIEW

File: hr_timesheet_activity_begin_end/README.rst [hunk 1]
```diff
@@ -21,30 +22,48 @@ Usage
 
 The new fields are displayed on the timesheet activities.
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/117/10.0
+
 Known issues / Roadmap
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/README.rst]: Remove this section as it's empty

File: hr_timesheet_activity_begin_end/__manifest__.py [hunk 2]
```diff
@@ -1,26 +1,9 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# Copyright 2015 Camptocamp SA - Guewen Baconnier
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/__manifest__.py]: Add your copyright

File: hr_timesheet_activity_begin_end/__manifest__.py [hunk 3]
```diff
@@ -1,26 +1,9 @@
...
 
 {'name': 'Timesheet Activities - Begin/End Hours',
- 'version': '8.0.1.0.0',
+ 'version': '10.0.1.0.0',
  'author': 'Camptocamp,Odoo Community Association (OCA)',
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/__manifest__.py]: Add Tecnativa

File: hr_timesheet_activity_begin_end/models/account_analytic_line.py [hunk 4]
```diff
@@ -1,31 +1,14 @@
...
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# Copyright 2015 Camptocamp SA - Guewen Baconnier
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/models/account_analytic_line.py]: Add your copyright

File: hr_timesheet_activity_begin_end/views/hr_analytic_timesheet.xml [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
+    <field name="arch" type="xml">
+      <field name="unit_amount" position="before">
+        <field name="time_start" widget="float_time"/>
+        <field name="time_stop" widget="float_time"/>
       </field>
-    </record>
+    </field>
+  </record>
+
+  <!--<record id="hr_timesheet_line_form" model="ir.ui.view">-->
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/views/hr_analytic_timesheet.xml]: Remove commented views

File: hr_timesheet_activity_begin_end/views/hr_timesheet_sheet.xml [hunk 6]
```diff
@@ -1,21 +1,20 @@
...
-        <xpath expr="//field[@name='timesheet_ids']/form/field[@name='unit_amount']" position="before">
-          <field name="time_start" widget="float_time"/>
-          <field name="time_stop" widget="float_time"/>
-        </xpath>
-      </field>
-    </record>
-  </data>
-</openerp>
+<odoo>
+  <record id="hr_timesheet_sheet_form" model="ir.ui.view">
```
Comment (pedrobaeza) [inline hr_timesheet_activity_begin_end/views/hr_timesheet_sheet.xml]: You are not using 4 spaces indent here

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by luismontalba** [comment]
Changes proposed done (commit whit changes squashed)

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu please review

**Comment by luismontalba** [comment]
After make a rebase apparently runbot image is built, but the link to data base seems to be broken:

enlace

**Comment by pedrobaeza** [comment]
Sometimes runbot doesn't complete the request, but just refresh and it's everything working.

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**